udata/tests/apiv2/test_topics.py

```diff
@@ -19,7 +19,8 @@ def test_topic_api_list(self):
         org = OrganizationFactory()
         paca, _, _ = create_geozones_fixtures()
 
-        tag_topic = TopicFactory(tags=["energy"])
+        tag_topic_1 = TopicFactory(tags=["my-tag-shared", "my-tag-1"])
+        tag_topic_2 = TopicFactory(tags=["my-tag-shared", "my-tag-2"])
```

Comment:
I'm confused, is this implemented only for the mongo search? I didn't see it in [the udata-search-service PR](https://github.com/opendatateam/udata-search-service/pull/50), am I missing something?

Reply:
Yup, mongo only. Topics are never handled by the search-service (unless I missed something huge :p) https://github.com/opendatateam/udata-search-service/blob/main/udata_search_service/domain/entities.py.

Reply:
I see, makes sense! Thanks 🙏